<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTrendChartPenDlg</class>
 <widget class="QWidget" name="QTrendChartPenDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>笔 #1</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P1_Min">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P1_Max">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线样式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜  色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="P1_Line">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="P1_Color">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>笔 #2</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P2_Min">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P2_Max">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线样式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜  色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="P2_Line">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="P2_Color">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>笔 #3</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P3_Min">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P3_Max">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线样式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜  色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="P3_Line">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="P3_Color">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>笔 #4</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P4_Min">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P4_Max">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线样式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜  色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="P4_Line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="P4_Color">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>笔 #6</string>
   </property>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P6_Min">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P6_Max">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="1000"/>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线样式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜  色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="P6_Line">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="P6_Color">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>笔 #5</string>
   </property>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P5_Min">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P5_Max">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="1000"/>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线样式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜  色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="P5_Line">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="P5_Color">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>笔 #7</string>
   </property>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P7_Min">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P7_Max">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线样式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜  色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="P7_Line">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="P7_Color">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>笔 #8</string>
   </property>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P8_Min">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="P8_Max">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线样式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜  色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="P8_Line">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="P8_Color">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>P1_Min</tabstop>
  <tabstop>P1_Max</tabstop>
  <tabstop>P1_Line</tabstop>
  <tabstop>P2_Min</tabstop>
  <tabstop>P2_Max</tabstop>
  <tabstop>P2_Line</tabstop>
  <tabstop>P3_Min</tabstop>
  <tabstop>P3_Max</tabstop>
  <tabstop>P3_Line</tabstop>
  <tabstop>P4_Min</tabstop>
  <tabstop>P4_Max</tabstop>
  <tabstop>P4_Line</tabstop>
  <tabstop>P5_Min</tabstop>
  <tabstop>P5_Max</tabstop>
  <tabstop>P5_Line</tabstop>
  <tabstop>P6_Min</tabstop>
  <tabstop>P6_Max</tabstop>
  <tabstop>P6_Line</tabstop>
  <tabstop>P7_Min</tabstop>
  <tabstop>P7_Max</tabstop>
  <tabstop>P7_Line</tabstop>
  <tabstop>P8_Min</tabstop>
  <tabstop>P8_Max</tabstop>
  <tabstop>P8_Line</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
